torch/_export/serde/serialize.py

```diff
@@ -939,6 +948,57 @@ def output_node_at_index(node, index):
 
         return output_arguments
 
+    def serialize_arbitrary_outputs(self, node: torch.fx.Node) -> List[Argument]:
+        """
+        For serializing HOO outputs since HOOs do not have a schema.
+        """
+        meta_val = node.meta["val"]
+
+        # Check single value return
+        if isinstance(meta_val, torch.Tensor):
+            # e.g "-> Tensor"
+            return [Argument.create(as_tensor=self.serialize_tensor_output(node.name, meta_val))]
+        elif isinstance(meta_val, torch.SymInt):
+            # e.g "-> SymInt"
+            return [Argument.create(as_sym_int=self.serialize_sym_int_output(node.name, meta_val))]
+        elif isinstance(meta_val, torch.SymBool):
+            # e.g "-> SymBool"
+            return [Argument.create(as_sym_bool=self.serialize_sym_bool_output(node.name, meta_val))]
+
+        # There are a two possibilities at this point:
```

Comment:
This implies that the only allowed outputs of a HOO are:
- Tensor
- Symint
- SymBool
- List[Tensor]
- Tuple[Tensor]

Is that right? Should we check and error on this explicitly? Right now I'm not sure what happens if we see a HOO that returns like `Tuple[Tensor, SymInt]`.

Reply:
Added an assert, although I think I should fix it so that we can return Tuple[Tensor, SymInt]